tests/test.hs

```diff
@@ -129,7 +131,8 @@ prop_alt_cont = once $
 
 prop_alt_help :: Property
 prop_alt_help = once $
-  let p = p1 <|> p2 <|> p3
+  let p :: Parser (Maybe (Either String String))
```

Comment:
I will probably not release this change immediately anyway, but wait until I can roll up a few more fixes into a release.